compat/surface_flinger/surface_flinger_compatibility_layer.cpp

```diff
@@ -330,13 +332,23 @@ SfSurface* sf_surface_create(SfClient* client, SfSurfaceCreationParameters* para
 		return NULL;
 	}
 
-	sf_client_begin_transaction(client);
-	{
-		surface->surface_control->setPosition(params->x, params->y);
-		surface->surface_control->setLayer(params->layer);
-		surface->surface_control->setAlpha(params->alpha);
-	}
-	sf_client_end_transaction(client);
+#if ANDROID_VERSION_MAJOR>=8
+    uint32_t mDisplayLayerStack = 0;
```

Comment:
Indentation issues (spaces vs tabs) here and many places below. This file seems to be using tabs.